tasks/ebpf_verifier/html/templates/index.html.j2

```diff
@@ -0,0 +1,58 @@
+<html>
+	<head>
+		<title>{{ title }}</title>
+		<link rel="preconnect" href="https://fonts.googleapis.com">
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+		<link href="https://fonts.googleapis.com/css2?family=Noto+Sans:ital,wght@0,100..900;1,100..900&display=swap" rel="stylesheet">
```

Comment:
```suggestion

```
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Is this really useful ? Some hosts network are behind whitelist so I'd avoid relying on google endpoints.

Reply:
It's mostly so we use the same fonts as the regular DataDog interface. Not really necessary, but on the other hand if the connection fails it's just going to fall back to another font. It's also going to render in dev's laptops, not on CI or any other kind of host.